src/index.js

```diff
@@ -17,30 +20,109 @@ if (!process.stdout.isTTY && !process.env.APPDATA) {
   process.exit(1);
 }
 
-const cookiesPath = process.pkg
-  ? path.join(path.dirname(process.execPath), "config", "cookies.json")
-  : "../config/cookies.json";
+// Function to spawn the browser
+function spawnBrowser(browserPath, debuggingPort) {
+  try {
+    console.log(`Attempting to launch browser: ${browserPath}`);
+    const args = [
+      `--remote-debugging-port=${debuggingPort}`,
+      `--user-data-dir=${profilePath}`
+    ];
+
+    const browserProcess = spawn(browserPath, args, {
+      detached: true,
+      stdio: "ignore",
+    });
+
```

Comment:
An event listener should be used to listen for errors on the browser process, if the BrowserPath does not exist this would "catch" that
```suggestion

    browserProcess.on("error", (error) => {
      // Probably ENOENT
      logError(`Failed to launch browser: ${error.message}`);
      console.log(
        `\nFailed to launch browser, check that your BrowserPath in config.json exists.`
      );
      process.exit(1);
    });
    
```